packages/RichTextEditing-Tests.package/RichTextEditorTextTests.class/instance/testCorrectLineColNumberAtArbitraryPositionInText.st

```diff
@@ -0,0 +1,8 @@
+unit-tests
+testCorrectLineColNumberAtArbitraryPositionInText
+
+	self 
+		enterStringOntoTextField: self multipleLineText;
```

Comment:
You enter the `multipleLineText` here but move the caret right based on the `standardText size`. Is that intended and why?

Reply:
Yea, kinda. The point is, that standardText is already entered in setup but the carret is still considerered to be on position 1. The multipleLineText is appended afterwards. Therefore, I have to move it by the size of stardardText first to get an senseful test result.